README.md
<p align="center">
    <img src="https://demo.phpmyjoomla.com/images/Logo_phpMyJoomla.png" alt="phpMyJoomla" width="350" height="350">
</p>

**Documentation:** www.phpmyjoomla.com

# Change Log
All notable changes to this project will be documented in this file.
This project adheres to [Semantic Versioning](http://semver.org/).

### Changed/Update
### Fixed
### Added
### Removed

## [3.0.0] - 2020-04
### Added
- INLINE EDITING.
- New, Edit and Delete buttons.
- Enable/Disable Inline Editing in configuration options.
- Enable/Disable accordion for Manage Tables view in configuration options.
- Help button added to toolbar.
- Spanish translation added to inline editing.
- Button tools used.

### Changed/Update
- FROM Font awesome 4.7.0. TO Font awesome 5.0.13.

### Fixed
- Alert when edit a table item deleted.
- Added a modal info after modify a table item.

### Removed
- Clean code

## [2.1.0] - 2020-03
### Added
- Excel and Print buttons.

### Fixed
- Change Log modal scrolling.
- Column visibility button fixed.

### Changed/Update
- Updated css files.
- Updated js files.

### Removed
- Clean code

## [2.0.1] - 2018-11
### Added
- Added plugin quick icon.
- Package plugin and component.

### Removed
- Clean code

## [2.0.0] - 2016-12
### Changed/Update
- Updated css files.
- Updated js files.

### Fixed
- bootstrap.min.js | removed becuase it is causing admin menu dropdowns to not work.

### Added
- Inline editing feature.
- Custom query feature.
- Font awesome 4.7.0.
- Added compatibility icons for joomla (3.x, 3.5, 3.4, 3.3) and php (7.0, 5.6, 5.5, 5.4).

### Removed
- Clean code

## [1.5.1] - 2016-01-11
### Changed/Update
- Updated js files.

## [1.5.0] - 2015-11-23
### Changed/Update
- Updated css files.
- Updated js files.

## [1.1.0] - 2015-04-06
### Added
- Spanish language.
- Chagelog file.
- Font awesome 4.3.0.

### Changed/Updated
- Changed icons.
- Updated JQuery 1.11.1 to JQuery 1.11.2.

## [1.0.0] - 2015-03-23
### Added
- Firt version 1.0.0 (Beta).


com_phpmyjoomla/CHANGELOG.md
## phpMyJoomla

**Documentation:** www.phpmyjoomla.com

# Change Log
All notable changes to this project will be documented in this file.
This project adheres to [Semantic Versioning](http://semver.org/).

### Changed/Update
### Fixed
### Added
### Removed

## [3.0.0] - 2020-04
### Added
- INLINE EDITING.
- New, Edit and Delete buttons.
- Enable/Disable Inline Editing in configuration options.
- Enable/Disable accordion for Manage Tables view in configuration options.
- Enable/Disable custom query in configuration options.
- Help button added to toolbar.
- Spanish translation added to inline editing.
- Button tools used.

### Changed/Update
- FROM Font awesome 4.7.0. TO Font awesome 5.13.0.

### Fixed
- Alert when edit a table item deleted.
- Added a modal info after modify a table item.

### Removed
- Clean code

## [2.1.0] - 2020-03
### Added
- Excel and Print buttons.

### Fixed
- Change Log modal scrolling.
- Column visibility button fixed.

### Changed/Update
- Updated css files.
- Updated js files.

### Removed
- Clean code

## [2.0.1] - 2018-11
### Added
- Added plugin quick icon.
- Package plugin and component.

### Removed
- Clean code

## [2.0.0] - 2016-12
### Changed/Update
- Updated css files.
- Updated js files.

### Fixed
- bootstrap.min.js | removed becuase it is causing admin menu dropdowns to not work.

### Added
- Inline editing feature.
- Custom query feature.
- Font awesome 4.7.0.
- Added compatibility icons for joomla (3.x, 3.5, 3.4, 3.3) and php (7.0, 5.6, 5.5, 5.4).

### Removed
- Clean code

## [1.5.1] - 2016-01-11
### Changed/Update
- Updated js files.

## [1.5.0] - 2015-11-23
### Changed/Update
- Updated css files.
- Updated js files.

## [1.1.0] - 2015-04-06
### Added
- Spanish language.
- Chagelog file.
- Font awesome 4.3.0.

### Changed/Updated
- Changed icons.
- Updated JQuery 1.11.1 to JQuery 1.11.2.

## [1.0.0] - 2015-03-23
### Added
- Firt version 1.0.0 (Beta).


script.pkg_phpmyjoomla.php
<?php
/**
 * @version     3.0.0
 * @package     phpMyJoomla
 * @copyright   Copyright (c) 2014-2020. Luis Orozco Olivares / phpMyjoomla. All rights reserved.
 * @license     GNU General Public License version 3 or later; see LICENSE.txt
 * @author      Luis Orozco Olivares <luisorozoli@gmail.com> - https://www.luisorozoli.com - https://www.phpmyjoomla.com
 */
// No direct access
defined('_JEXEC') or die;

class Pkg_phpMyJoomlaInstallerScript
{

	public function install ($parent)
	{
	}

	public function update ($parent)
	{
	}

	public function uninstall ($parent)
	{
	}

	public function postflight ($type, $parent)
	{
		$status = new stdClass;
		$this->installationResults($status);
	}

	private function installationResults ($status)
	{
		$language = JFactory::getLanguage();
		$language->load('com_phpmyjoomla');
		$rows = 0;
		?>
		<div>
			<a href="https://www.phpmyjoomla.com" target="_blank"><img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/Logo_phpMyJoomla.png" alt="phpMyJoomla logo" align="left" style="max-width: 200px;"></a>
			<a href="https://www.luisorozoli.com" target="_blank"><img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/logo_luisorozoli.png" alt="Luisorozoli logo" align="right" style="max-width: 200px;"></a>
		</div>
		<div align="center">
			<h1>phpMyJoomla Installation</h1>
			<h3><a href="index.php?option=com_phpmyjoomla">Go to the component phpMyJoomla</a></h3>
			<h3><a href="index.php?option=com_plugins&view=plugins&filter_search=phpmyjoomla">ENABLE the Pluging Quickicon phpMyJoomla</a></h3>
		</div>
		<br/>

		<div>
			<h3>TOOLS USED</h3>
			<p>
				For the development of this component have been used the following tools:<br /><br />
				<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/joomla_3x.png" alt="Joomla! 3.x" align="left" style="padding-right: 5px;" />
				<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/joomla_35.png" alt="Joomla! 3.5" align="left" style="padding-right: 5px;" />
				<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/joomla_34.png" alt="Joomla! 3.4" align="left" style="padding-right: 5px;" />
				<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/joomla_33.png" alt="Joomla! 3.3" align="left" style="padding-right: 5px;" />
				<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/php-70x.png" alt="PHP 7.0" align="left" style="padding-right: 5px;" />
				<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/php-56x.png" alt="PHP 5.6" align="left" style="padding-right: 5px;" />
				<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/php-55x.jpeg" alt="PHP 5.5" align="left" style="padding-right: 5px;" />
				<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/php-54x.jpeg" alt="PHP 5.4" align="left" />
				<a href="https://extensions.joomla.org/extension/phpmyjoomla/" target="_blank" style="float: right;">
					<img src="components/com_phpmyjoomla/assets/images/phpmyjoomla/joomla-extdev.png" alt="phpMyJoomla JED" align="left" style="max-width: 200px;">
				</a>
			</p>
		</div>
		<br/>

		<div>
			<p>
				- <a href="https://component-creator.com/" target="_blank">Component Creator</a><br />
				- <a href="https://datatables.net/" target="_blank">DataTables</a><br />
				- <a href="https://getbootstrap.com/" target="_blank">Bootstrap</a><br />
				- <a href="https://fontawesome.com/" target="_blank">Font Awesome</a><br />
				- <a href="https://jquery.com/" target="_blank">JQuery & Ajax</a>
			</p>
		</div>
		<br/>

		<div class="">
			<h3>MEMBERS & SUPPORT</h3>
			<div>
				<p>
					Support in English and Spanish!
				</p>
				<p class="">
					<a href="https://www.phpmyjoomla.com/member-login" target="_blank"><button type="button" class="btn btn-success"><i class="icon-users" style="padding-right: 10px;"></i>MEMBERS</button></a>
					<a href="https://www.phpmyjoomla.com/support" target="_blank"><button type="button" class="btn btn-success"><i class="icon-question-sign" style="padding-right: 10px;"></i>SUPPORT</button></a>
					<a href="https://www.phpmyjoomla.com" target="_blank"><button type="button" class="btn btn-info"><i class="icon-book" style="padding-right: 10px;"></i>DOCUMENTATION</button></a>
				</p>
				<br/>
				<h3>Our Location</h3>
				<p>
					Luis Orozco Olivares | <a href='https://www.luisorozoli.com' target="_blank">www.luisorozoli.com</a> | Jaén (Spain) | Spanish & English.
				</p>
				<br/>
			</div>

			<h3>More information about the component in <a href='https://www.phpmyjoomla.com' target="_blank">www.phpmyjoomla.com</a>.</h3><br />
		<?php
	}

}
